packages/blocks/src/api/raw-handling/index.js

```diff
@@ -190,7 +191,9 @@ export default function rawHandler( { HTML = '', plainText = '', mode = 'AUTO',
 					'Sanitized HTML: `' + node.outerHTML + '`'
 				);
 
-				return;
+				return createBlock( 'core/html', {
+					content: originalPieceFiltered,
```

Comment:
This isn't quite right. We will want to return the HTML of the current _node_ here, not the entire _piece_.

```js
return createBlock( 'core/html', {
	content: node.outerHTML,
} );
```

You can see this bug by:

1. Create a new post
2. Run `wp.blocks.unregisterBlockType( 'core/preformatted' )` in the console
3. Copy the **entire** text of this comment including the code snippet above
4. Paste it into Gutenberg

Only the code snippet above should appear as a HTML block. Instead, _the entire comment_ appears in the HTML block.

Reply:
GIF of the above:

![bug](https://user-images.githubusercontent.com/612155/48042762-3cdaf180-e1d7-11e8-9c1d-ae545860d419.gif)